pulseapi/profiles/serializers.py

```diff
@@ -96,5 +98,13 @@ def get_published_entries(self, instance):
 
     my_profile = serializers.SerializerMethodField()
 
+    def get_created_entries(self, instance):
+        entry_creator_records = OrderedCreatorRecord.objects.filter(creator__profile=instance)
+        entries = []
```

Comment:
Python has ways.
```
entries = list(map(lambda x: EntrySerializer(x.entry).data, entry_creator_records))
```

Reply:
This works, but you wanna give me the tl;dr (the documentation) on what it's doing?